src/components/Queue.vue

```diff
@@ -16,6 +16,8 @@
             </div>
         </transition>
 
+        <div v-on:click="collapsed = !collapsed" class="expandable queue" v-bind:class="{'expanded' : !collapsed}"><i class="fa fa-list" aria-hidden="true"></i></div>
```

Comment:
As per my comment to @ob6160 we seem to sometimes be using v-on and v-bind and sometimes using @ and :

Either fine - but should be consistent. I think we are mostly being explicit.